src/ui/pages/Receive/AddressDetails.tsx

```diff
@@ -1,13 +1,200 @@
-import React from 'react';
+import React, { useMemo, useRef } from 'react';
 import { QRCode } from 'react-qrcode-logo';
 import { Button } from 'src/ui/ui-kit/Button';
 import { HStack } from 'src/ui/ui-kit/HStack';
-import { Spacer } from 'src/ui/ui-kit/Spacer';
 import { UIText } from 'src/ui/ui-kit/UIText';
 import { VStack } from 'src/ui/ui-kit/VStack';
 import CopyIcon from 'jsx:src/ui/assets/copy.svg';
 import CheckIcon from 'jsx:src/ui/assets/check_double.svg';
 import { useCopyToClipboard } from 'src/ui/shared/useCopyToClipboard';
+import { UnstyledButton } from 'src/ui/ui-kit/UnstyledButton';
+import { BottomSheetDialog } from 'src/ui/ui-kit/ModalDialogs/BottomSheetDialog';
+import { DialogTitle } from 'src/ui/ui-kit/ModalDialogs/DialogTitle';
+import type { HTMLDialogElementInterface } from 'src/ui/ui-kit/ModalDialogs/HTMLDialogElementInterface';
+import EcosystemSolanaIcon from 'jsx:src/ui/assets/ecosystem-solana.svg';
+import EcosystemEthereumIcon from 'jsx:src/ui/assets/ecosystem-ethereum.svg';
+import { useNetworks } from 'src/modules/networks/useNetworks';
+import { PageStickyFooter } from 'src/ui/components/PageStickyFooter';
+import { isCustomNetworkId } from 'src/modules/ethereum/chains/helpers';
+import { NetworkIcon } from 'src/ui/components/NetworkIcon';
+import {
+  getAddressType,
+  type BlockchainType,
+} from 'src/shared/wallet/classifiers';
+
+function NetworkList({ standard }: { standard: BlockchainType }) {
+  const { networks } = useNetworks();
+  const allNetworks = useMemo(() => {
+    return networks
+      ?.getDefaultNetworks(standard)
+      .filter((item) => !item.is_testnet && !isCustomNetworkId(item.id));
+  }, [networks]);
+
+  return (
+    <VStack gap={0}>
+      {allNetworks?.map((network) => (
+        <HStack key={network.id} gap={12} style={{ paddingBlock: 12 }}>
+          <NetworkIcon name={network.name} src={network.icon_url} size={24} />
+          <UIText kind="body/regular">{network.name}</UIText>
+        </HStack>
+      ))}
+    </VStack>
+  );
+}
+
+function EthereumNetworksVisualStack() {
+  return (
+    <div style={{ display: 'flex' }}>
+      <img
```

Comment:
do we need alt here?
and should we extract the same style as a separate var?

Reply:
Yes, thx! I was thinking about ways to abstract this and ended up skipping it :)